internal/osutils/privileges_win.go

```diff
@@ -1,12 +1,22 @@
+//go:build windows
 // +build windows
 
 package osutils
 
 import (
+	"os"
+
+	"github.com/ActiveState/cli/internal/constants"
 	"golang.org/x/sys/windows"
 )
 
+const EnvOverrideAdmin = "ACTIVESTATE_CLI_ISADMIN_OVERRIDE"
+
 func IsAdmin() (bool, error) {
+	// Work around weird CI bug: DX-1329
```

Comment:
Would you please put a one-sentence summary so I or anyone else does not have to search JIRA for this ticket?